app/models/vmdb_database/seeding.rb

```diff
@@ -42,11 +41,16 @@ def db_server_ipaddress
       host
     end
 
-    def db_disk_size(disk)
-      MiqSystem.disk_usage(disk).first[:filesystem]
```

Comment:
haha, oops... I guess we're using the disk usage as a side effect for checking that the disk is mounted.  But yeah, that name for this method is bad.